services/s3/src/main/java/software/amazon/awssdk/services/s3/internal/crt/DefaultS3CrtAsyncClient.java

```diff
@@ -280,6 +283,15 @@ public S3CrtAsyncClient build() {
     }
 
     private static final class AttachHttpAttributesExecutionInterceptor implements ExecutionInterceptor {
+
+        @Override
+        public void beforeExecution(Context.BeforeExecution context, ExecutionAttributes executionAttributes) {
+            // Hack to disable new SRA path because we still rely on HttpChecksumStage to perform checksum for
+            // non-streaming operation.
+            // TODO: remove this once CRT supports checksum for default requests
```

Comment:
@zoewangg @cenedhryn what does "default requests" here mean? Does it mean non-streaming requests?

Reply:
Yes